src/xmipp/libraries/reconstruction_adapt_cuda/xmipp_gpu_correlation.cpp

```diff
@@ -1605,10 +1572,9 @@ void ProgGpuCorrelation::run()
 				max_vector_rt_mirror[i]=-1;
 			}
 
-			expIndex = iterExp->objId;
 			available_images_exp--;
 
-			SF.getRow(rowExp, expIndex); //SFexp
+			MDRow& rowExp = *iterExp;
```

Comment:
Is this equivalent?

Reply:
Yes, dereferencing row iterator returns row :)